redash/query_runner/athena.py

```diff
@@ -78,6 +78,11 @@ def configuration_schema(cls):
                     'type': 'boolean',
                     'title': 'Use Glue Data Catalog',
                 },
+                'work_group': {
```

Comment:
Could you add this to the correct place in `order`? (it happens now automatically due to the alphabetical order, but who knows what will come next)